libs/application/templates/signature-collection/parliamentary-list-creation/src/lib/constants.ts

```diff
@@ -0,0 +1,46 @@
+import { DefaultEvents } from '@island.is/application/types'
+
+export type Events = { type: DefaultEvents.SUBMIT }
+
+export enum States {
+  DRAFT = 'draft',
+  DONE = 'done',
+  PREREQUISITES = 'PREREQUISITES',
+}
+export enum Roles {
+  APPLICANT = 'applicant',
+}
+
+export type SubmitResponse = {
+  success: boolean
+  orderId?: string[]
+}
+
+export enum ApiActions {
+  submitApplication = 'createLists',
+}
+
+export type Manager = {
+  manager: {
+    name: string
+    nationalId: string
+  }
+  constituency: string
+}
+
+export type Supervisor = {
+  supervisor: {
+    name: string
+    nationalId: string
+  }
+  constituency: string
+}
+
+export const Constituencies = [
+  'Norðausturkjördæmi',
+  'Norðvesturkjördæmi',
+  'Reykjavíkurkjördæmi norður',
+  'Reykjavíkurkjördæmi suður',
+  'Suðurkjördæmi',
+  'Suðvesturkjördæmi',
+]
```

Comment:
These will be fetched later? Should there be a comment above explaining this as a note, just in case someone else will have to maintain this?